central/graphql/resolvers/loaders/nodes.go

```diff
@@ -58,16 +60,38 @@ type nodeLoaderImpl struct {
 
 // FromIDs loads a set of nodes from a set of ids.
 func (ndl *nodeLoaderImpl) FromIDs(ctx context.Context, ids []string) ([]*storage.Node, error) {
-	nodes, err := ndl.load(ctx, ids)
+	nodes, err := ndl.load(ctx, ids, false)
 	if err != nil {
 		return nil, err
 	}
 	return nodes, nil
 }
 
-// FromID loads a node from an ID.
+// FromID loads a node from an ID. Does not load scan components and vulnerabilities when Postgres is enabled.
 func (ndl *nodeLoaderImpl) FromID(ctx context.Context, id string) (*storage.Node, error) {
-	nodes, err := ndl.load(ctx, []string{id})
+	nodes, err := ndl.load(ctx, []string{id}, false)
```

Comment:
It would be helpful if we have metrics around load cache here. But this is out of scope for this PR.

Reply:
Yeah, it would be useful to know especially memory usages, to see if the cache takes up too much space.